docs/env-definitions.md

```diff
@@ -10,7 +10,7 @@ If you're on version `>=3.8.0` you can make use of the `env` property in [flow-t
 
 ```json
 {
-  env: ["jsx", "node", ...],
+  "env": ["jsx", "node", ...],
```

Comment:
See discussion of the `...` being red in here:

* https://github.com/flow-typed/flow-typed/pull/4515#discussion_r1329916073